packages/snowpack/src/commands/build.ts

```diff
@@ -218,7 +218,8 @@ export async function command(commandOptions: CommandOptions) {
 
       // We treat ".proxy.js" files special: we need to make sure that they exist on disk
       // in the final build, so we mark them to be written to disk at the next step.
-      const isAbsoluteUrlPath = resolvedImportUrl.startsWith('/');
+      const isAbsoluteUrlPath =
+        resolvedImportUrl.startsWith('/') || resolvedImportUrl.startsWith('\\'); // note: Windows backslashes may still be present at this point
```

Comment:
Windows bug with aliases: we store POSIX-style forward slashes in `import-map.json`, which is good for consistency but causes problems when we try and resolve web modules using a mixture of forward- and backslashes. Node’s `path` fails when it encounters both.

So the fix is we have to do a little bit of back-and-forth: on Windows we resolve using backslashes, then replace the output to forward slashes at the very end. We can’t do this earlier as we don’t know the relative location from each file.

Reply:
if `resolvedImportUrl` is a file system path, then we can use `path.isAbsolute` as the Node-approved way to check for this, cross-env

Reply:
TIL